test/integration/scenarios/05.catalog/catalogs/config.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: federation.symphony/v1
+kind: Catalog
+metadata:
+  name: config
+spec:  
+  siteId: hq
+  type: config
+  name: config
+  metadata:
```

Comment:
We need to restore the Metadata field under the CatalogSpec struct

Reply:
https://github.com/eclipse-symphony/symphony/issues/196
Created an issue for this. And I am working on the issue now. 
I've commented the schema cases. Can we merge the test code first?